Sources/SWBCore/Core.swift

```diff
@@ -169,8 +174,26 @@ public final class Core: Sendable {
 
     public let pluginManager: PluginManager
 
+    public enum DeveloperPath: Sendable, Hashable {
```

Comment:
Yeah, this is great, our previous model definitely didn't well capture the idea that we need to support:
- macOS with Xcode only
- macOS with swift.org toolchain + DCLT
- macOS with swift.org toolchain + Xcode